librocksdb-sys/build.rs

```diff
@@ -198,6 +198,11 @@ fn build_rocksdb() {
         config.define("WITH_JEMALLOC", "ON");
     }
 
+    if target.contains("linux") && cfg!(feature = "io-uring") {
+        pkg_config::probe_library("liburing").unwrap();
```

Comment:
What will be if there is no `liburing` library on a host?

```suggestion
        if pkg_config::probe_library("liburing").is_ok() {
            config.define("ROCKSDB_IOURING_PRESENT", Some("1"));
        }
```